narwhals/_dask/expr.py

```diff
@@ -663,6 +663,12 @@ def is_finite(self) -> Self:
 
         return self._with_callable(da.isfinite, "is_finite")
 
+    def first(self) -> Self:
+        def fn(_input: dx.Series) -> dx.Series:
+            return _input[0].to_series()
```

Comment:
I am getting an error along the lines of: "series has no method `to_series`"

```suggestion
            return _input[0]
```

Reply:
Also, shouldn't we be using `iloc` instead?

Reply:
@FBruzzesi happy to use whatever you find works 🙌

I was basing this off of some other aggregation implementation, will need to dig for which one

Reply:
>I was basing this off of some other aggregation implementation , will need to dig for which one

I think that's because in those cases a dask Expr is returned.

By checking the docs, there is no `iloc` for [series](https://docs.dask.org/en/stable/generated/dask.dataframe.Series.html)?!

Maybe we can do:

```suggestion
            return _input.head(1)
```

instead 🤔

Reply:
(https://github.com/narwhals-dev/narwhals/pull/2528/commits/bb9912d76927b527962d400159549be8328a094e)

@FBruzzesi yeah I had that thought too - whatever you think I'm happy with

> - Less sure about this one
> - `head(1)` also seemed like an option

Reply:
I'm getting a bunch of errors for `head(1)`, should show up in (https://github.com/narwhals-dev/narwhals/pull/2528/commits/77ae9c0300efa126a39bf5812832397e32fc4394):

```py
E           AttributeError: 'NFirst' object has no attribute 'sort'
E
E           This often means that you are attempting to use an unsupported API function..

.venv/Lib/site-packages/dask/_expr.py:820: AttributeError
```

```py
df = 0    1.0
1    1.0
Name: c, dtype: float64, columns = ['idx'], n = 1, ascending = True

    def _nfirst(df, columns, n, ascending):
>       return df.sort_values(by=columns, ascending=ascending).head(n)
E       TypeError: Series.sort_values() got an unexpected keyword argument 'by'

.venv/Lib/site-packages/dask/dataframe/dask_expr/_reductions.py:1377: TypeError
```